Frontend/UserTech/src/app/app-routing.module.ts

```diff
@@ -28,7 +28,7 @@ const routes: Routes = [
         component: HomeComponent,
       },
       {
-        path: 'station/:id',
+        path: 'station/:id/:name',
```

Comment:
Nie podoba mi się to mocno, bo ktoś może zmienić nazwę i dalej będzie ta sama stacja, ale nw też jakby to inaczej rozwiązać.

Reply:
Widziałem że jest jakaś biblioteka która pozwala na przekazywanie czegoś routerLinkiem nie przez URL, ale wtedy przy wpisaniu urlu nie pojawi się wogóle tytuł